chore: add postgres startup script (#54)
openfga/openfga: README.md

./openfga run
```

### Running with Postgres

This section assumes that you have cloned the repository.

To run OpenFGA with the Postgres datastore engine, simply run the following commands:
```bash
docker compose up -d postgres
make run-postgres
```

> If everything works, this should start a Postgres container, write the schema, and start the OpenFGA server backed by the Postgres instance.
> 
> When you are done you can stop the container with:
> ```bash
> docker compose down
> ```

Reviewer: ```suggestion
This should start a Postgres container, write the database schema, and start the OpenFGA server.

When you are done you can stop the Postgres container with:
```bash
docker compose down
```
```

Author: Thanks!
